mailpoet.php

```diff
@@ -31,7 +31,7 @@ function mailpoet_deactivate_plugin() {
 }
 
 // Check for minimum supported WP version
-if (version_compare(get_bloginfo('version'), '4.6', '<')) {
+if (version_compare(get_bloginfo('version'), '5.3', '<')) {
```

Comment:
I think we should be more conservative here. We still have a bunch of users with 5.2 < who updates. WP itself shows some warnings.
The plugin should still work 5.0+ so I would go with 5.0 as a first step